code/game/objects/structures/crates_lockers/closets/utility_closets.dm

```diff
@@ -30,12 +30,13 @@
 		if ("small")
 			new /obj/item/tank/internals/emergency_oxygen(src)
 			new /obj/item/tank/internals/emergency_oxygen(src)
+			new /obj/item/storage/firstaid/emergency(src)
 			new /obj/item/clothing/mask/breath(src)
 			new /obj/item/clothing/mask/breath(src)
 
 		if ("aid")
 			new /obj/item/tank/internals/emergency_oxygen(src)
-			new /obj/item/storage/firstaid/o2(src)

```

Comment:
aw, I liked the o2 first aid kits, they were a great source of salb pens

maybe you could add a salb pen to the emergency first aid kit as compensation?